playwright.config.ts

```diff
@@ -101,10 +101,10 @@ const config: PlaywrightTestConfig = {
   // outputDir: 'test-results/',
 
   /* Run your local dev server before starting the tests */
-  // webServer: {
-  //   command: 'npm run start',
-  //   port: 3000,
-  // },
+  webServer: {
+    command: 'npm run start',
+    port: 8083,
```

Comment:
Why `8083`? Where is this configured? For me, `8080` works.
Could it be that Webpack chooses another port if the default port is taken and you happened to have other servers running while testing this?